doc/widgets/networkfile.md

```diff
@@ -0,0 +1,28 @@
+Network File
+============
+
+Read network graph file in Pajek or GML format.
```

Comment:
@janezd Is this still correct? I think we atm support only Pajek.

Reply:
Yes, we support only Pajek.